content/audio.md

```diff
@@ -144,79 +186,38 @@ If you would like to monitor PipeWire, run:
 pw-top
 ```
 
-## Configuration Tweaks
-
-Some particular problems may be solved by tweaks to ALSA or PulseAudio configuration. Clearing the current settings for Pipewire or PulseAudio may allow the defaults to be used again. To revert to defaults and clear any current saved settings run the following commands:
-
-```
-rm -r ~/.config/pulse/*
-rm -r ~/.local/state/wireplumber/*
-```
-
-**NOTE:** When running those commands you may see messages similar to this: `rm: cannot remove '/home/ckw/.config/pulse/*': No such file or directory` those are safe to ignore as the file simply does not exist.
+## Installing Sound Open Firmware Binaries
 
-### Audio crackling or hardware clicking
+This firmware is essential for proper audio functionality on systems using Intel audio hardware that relies on Sound Open Firmware.
 
-If you hear audio crackling (especially when you start or stop playing audio), your audio card may be going to sleep too often. This is known to happen on some versions of the [Serval WS](/articles/serval-dac/) and some [Thunderbolt docks](https://github.com/system76/docs/issues/491).
+To install, run the command:
```

Comment:
`commands` should be plural if you're going to include the update/upgrade commands first. (The `upgrade` command isn't strictly on-topic for these code blocks.)